python/tvm/relax/dpl/pattern.py

```diff
@@ -1115,3 +1115,30 @@ def make_fused_bias_activation_pattern(op_name, with_bias=False, activation=None
         return is_op(activation)(out)
 
     return out
+
+
+def rewrite(
+    pattern: DFPattern, callback: Callable[[Expr, Dict[DFPattern, Expr]], Expr], func: Function
```

Comment:
It actually took me some time to understand how `callback` works and can be written. Not sure if there are better ways to simplify it. But it could be just me.

Reply:
Replaced `callback` to `rewriter`. 

Also added an example of a simple rewriter in the doc.